cmd/controller/main.go

```diff
@@ -78,6 +78,7 @@ func main() {
 			Allocator: &allocation.GreedyAllocator{
 				Capacity: cloudProviderFactory.Capacity(),
 			},
+			ProcessedPods: make(map[string]bool),
```

Comment:
I want to avoid bloating main. Can we encapsulate this in a constructor for the controller?

Reply:
Kept this as is for now, this will be removed once we make changes for cluster state or pod binding.